web_src/js/features/tablesort.js

```diff
@@ -0,0 +1,29 @@
+export default function initTableSort() {
+  $('th[data-sortt-asc]').each(function () {
+    // get data
+    const {sorttAsc, sorttDesc, sorttDefault} = this.dataset;
+
+    // add onclick event
+    $(this).on('click', () => {
+      tableSort(sorttAsc, sorttDesc, sorttDefault);
+    });
+  });
```

Comment:
```suggestion
  for (const header of document.querySelectorAll('th[data-sortt-asc]')) {
    const {sorttAsc, sorttDesc, sorttDefault} = header.dataset;
    header.addEventListener('onclick', () => {
      tableSort(sorttAsc, sorttDesc, sorttDefault);
    });
  }
```

We should reduce jQuery usage and obvious comments add no value so I removed them.